src/components/SearchBox.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+class SearchBox extends React.PureComponent {
+	state = {
```

Comment:
As per docs, you should only assign state in constructor.
https://reactjs.org/docs/state-and-lifecycle.html#adding-local-state-to-a-class:~:text=The%20only%20place%20where%20you%20can%20assign%20this.state%20is%20the%20constructor.